src/app/exception_throw.py

```diff
@@ -0,0 +1,94 @@
+from fastapi import status
+from fastapi.responses import JSONResponse
+
+
+class exception_response:
```

Comment:
You should be following class naming conventions i.e. Pascal Case, so this should be ExceptionResponse. Also I don't think this is the best name for the class either, I'd call it maybe ExceptionResponder. Also the class name and the file name should be consistent, when importing this class you should probably be doing e.g.

from exception_responder import ExceptionResponder

Reply:
Changed the name to follow the naming conventions